pkgs/development/python-modules/torch/bin.nix

```diff
@@ -55,6 +67,13 @@ in buildPythonPackage {
   postFixup = let
     rpath = lib.makeLibraryPath [ stdenv.cc.cc.lib ];
   in ''
+    pushd $out/${python.sitePackages}/torch/lib
+      LIBNVRTC=`ls libnvrtc-* |grep -v libnvrtc-builtins`
+      if [ ! -z "$LIBNVRTC" ] ; then
+        ln -s "$LIBNVRTC" libnvrtc.so
+      fi
+    popd
```

Comment:
Maybe we remove this altogether and instead use `autoPatchelf` with `cuda_nvrtc` in `buildInputs`. The libnvrtc-builtins won't be an issue once https://nixpk.gs/pr-tracker.html?pr=226038 reaches master